### PR TITLE
Change cosmos sample from t1 to t2

### PR DESCRIPTION
**Pull request by Melanie-Ma**
Change cosmos sample from track1 to track2.

@jongio, for notification.

### CODE REVIEW

**Comment by jongio** [comment]
@conniey- Can you please review this?  Thanks

File: pom.xml [hunk 1]
```diff
@@ -11,17 +11,25 @@
...
   <dependencies>
-  <dependency>
-    <groupId>com.microsoft.azure</groupId>
-    <artifactId>azure-documentdb</artifactId>
-    <version>LATEST</version>
-</dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+      <version>4.5.0</version>
```
Comment (helayoty) [inline pom.xml:32]: Would you please change the version to use the latest?

File: src/GetStarted/Program.java [hunk 2]
```diff
@@ -2,63 +2,96 @@
...
+    private CosmosClient client;
+    private CosmosDatabase database;
+    private CosmosContainer container;
+
+    private final String databaseName = "FamilyDB";
+    private final String collectionName = "FamilyCollection";
 
-    private DocumentClient client;
-
-    /**
```
Comment (helayoty) [inline src/GetStarted/Program.java:24]: Please undo the delete of the main function comment.

**Review by helayoty — CHANGES_REQUESTED** [review]
Thank you @Melanie-Ma for your PR. I've added a few comments.

**Review by kushagraThapar — CHANGES_REQUESTED** [review]
The repo name is outdated. Also we already have samples for track 2 library here : https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples
I am not sure what is the purpose of these samples?

**Comment by jongio** [comment]
@kushagraThapar - Would you like to make the call to delete this sample or remove the code and point to the replacement sample?

**Comment by kushagraThapar** [comment]
@jongio - I am fine with both deleting / removing the code and pointing to the new sample. Ideally we should delete it, but I am fine keeping it as well (since the azure-documentdb SDK is still not deprecated) - but we should definitely remove the code from this sample, and point to the new samples.

**Comment by jongio** [comment]
@kushagraThapar - Can you please provide a link to the sample you want to point to?  We'll update the readme with that link and then delete all the code here.  Thanks.

**Comment by kushagraThapar** [comment]
@jongio - this is the getting-started guide we would like to point to - https://github.com/Azure-Samples/azure-cosmos-java-getting-started
Also, we can link the samples as well - https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks!

**Review by jongio — APPROVED** [review]

**Pull request merged**